package.json

```diff
@@ -44,6 +44,8 @@
   "dependencies": {
     "@babel/core": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
+    "@types/pino": "^6.3.2",
+    "@types/pino-http": "^5.0.5",
```

Comment:
types normally go in `devDependencies`

Reply:
Good spot! Have moved them in the latest commit 👍